frontend/packages/vjs-plugin/src/styles.scss

```
@@ -865,7 +865,7 @@ $controlBarAnimationsDuration: 0.1s;
 		width: $control-bar-button-min-width;
 		line-height: $control-bar-height - 2px;
 
-		text-shadow: 0 1px 3px rgba(#000, 0.05);
+		text-shadow: 0 1px 3px rgba(var(--cinemata-black), 0.05);
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)